spec/components/candidate_interface/english_foreign_language/ielts_review_component_spec.rb

```diff
@@ -35,8 +65,13 @@
     )
     result = render_inline(described_class.new(ielts_qualification, return_to_application_review: true))
```

Comment:
this line can be deleted (and the linter won't be happy until it is)

Reply:
Thanks. Missed that one! 🤦